Library/Formula/fish-tank.rb

```diff
@@ -0,0 +1,15 @@
+class FishTank < Formula
+  homepage "https://github.com/terlar/fish-tank"
+  url "https://github.com/terlar/fish-tank/archive/0.1.1.tar.gz"
+  sha1 "b0be92bf01dae05e0efced363139d0d813365485"
+
+  head "https://github.com/joelthelion/autojump.git"
+
+  def install
+    system "make", "install"
```

Comment:
This formula will need to install to the prefix defined by `prefix`, which will be a path like `/wherever/homebrew/is/installed/Cellar/fish-tank/1.2.3`. Looking at the Makefile, `system "make", "install", "PREFIX=#{prefix}"` is probably the right incantation.